molgenis-data-mapper/src/main/resources/js/attribute-mapping.js

```diff
@@ -274,7 +355,10 @@
 			// update algorithm
 			algorithm = editor.getSession().getValue();
 
-			// update result
+			// validate mapping
+			validateAttrMapping(algorithm);
```

Comment:
The map tab being changed will update the result, but will not activate the validateAttrMapping. This is not a problem for now but we should remember it for future reference